.github/workflows/main.yaml

```diff
@@ -257,6 +262,16 @@ jobs:
         uses: ./.github/actions/gn
       - name: Build Cobalt
         uses: ./.github/actions/build
+      - name: 'Check Artifact Size'
+        uses: ./.github/actions/check_artifact_size
+        if: ${{ inputs.artifact_size_increase_percentage_thresholds }}
+        continue-on-error: true  # Ignore this step if check artifact size failed.
```

Comment:
This may be dangerous. If this step starts failing we won't see that anywhere unless we manually check the logs. IMO we want to know if e.g. an API call starts failing or if there's a syntax error in the configuration.

Reply:
I’ve been considering which scenarios should cause the action to fail.
But in this PR we can not error out this step because I didn't get the configuration right...
I forgot to change the artifact path to install folder in the previous upload PR and have to adjust them again in this PR: [line](https://github.com/youtube/cobalt/pull/3813/files#diff-6fc52dea1fadeb587d62a84233db8977082dbd4d8767378f00b7a05067e679c8R57.)

(I was too stressed from trying to get method 1 to work, and overlooked the need to change to install path when I switched to the method 2 in the previous upload PR [reference] (https://b.corp.google.com/issues/291954423#comment4)).

Can we merge this as is, to verify if add-label works and add failing condition in a followup PR?

Reply:
Agreed offline to remove in follow-up PR.